cpp/pybind/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ elseif (UNIX)   # Linux
 };]=])
     target_link_options(pybind PRIVATE $<$<CONFIG:Release>:
         "-Wl,--version-script=${CMAKE_CURRENT_BINARY_DIR}/pybind.map" >)
+    target_link_options(pybind PRIVATE "-flto=auto")
```

Comment:
Hi @roehling Is this something we can have for the Open3D DSO as well? (in `cpp/open3d/CMakeLists.txt`)

Reply:
Yes, but only after #4747 has been resolved.